pacman/PKGBUILD

```diff
@@ -1,7 +1,7 @@
 pkgname=onedrive
 pkgver=1.1.3
```

Comment:
Should this not need to be `pkgver=2.1.5` which is the current version of this fork?